presto-main/src/main/java/com/facebook/presto/sql/rewrite/ShowStatsRewrite.java

```diff
@@ -150,12 +152,11 @@ else if (node.getRelation() instanceof Table) {
             }
         }
 
-        private void validateShowStats(ShowStats node)
+        private void validateShowStats(ShowStats node, Plan plan)
```

Comment:
`if (node.getRelation() instanceof TableSubquery) {` on L167 is redundant as this method is only called if it is true